pkgs/by-name/cl/cli-tips/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  bash,
+  makeWrapper,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "cli-tips";
+  version = "0-unstable-2024-11-14";
+
+  src = fetchFromGitHub {
+    owner = "cli-stuff";
+    repo = "cli-tips";
+    rev = "ebc191a54be7e39accd1948c3de8aded438d0495";
+    sha256 = "sha256-KELgatdL+2M5ktuAHEljIEJ9wqP578dp5tYWYPpP3bg=";
+  };
+
+  buildInputs = [ bash ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    mkdir -p $out/bin $out/share
+    cp cli-tips.sh $out/bin/cli-tips
+    cp -r translations $out/share
+    chmod +x $out/bin/cli-tips
```

Instead of copying and chmod-ing, just `install` it in one step. Also, this should be in `installPhase` and not `configurePhase`
```suggestion
    mkdir -p $out/share
    install -Dm755 cli-tips.sh $out/bin/cli-tips
    cp -r translations $out/share
```